src/jdk.accessibility/windows/native/libwindowsaccessbridge/AccessBridgeJavaVMInstance.cpp

```diff
@@ -268,7 +268,7 @@ AccessBridgeJavaVMInstance::sendMemoryPackage(char *buffer, long bufsize) {
             DEBUG_CODE(PackageType *type = (PackageType *) memoryMappedView);
             DEBUG_CODE(if (*type == cGetAccessibleTextItemsPackage) {)
                 DEBUG_CODE(AppendToCallInfo("  'memoryMappedView' now contains:"));
-                DEBUG_CODE(GetAccessibleTextItemsPackage *pkg = (GetAccessibleTextItemsPackage *) (buffer + sizeof(PackageType)));
+             // DEBUG_CODE(GetAccessibleTextItemsPackage *pkg = (GetAccessibleTextItemsPackage *) (buffer + sizeof(PackageType)));
```

Comment:
Why do we need to comment out that? In non-debug build it will be replaced by the comment block anyways.

Reply:
In release builds this isn't an issue, but when building either fastdebug or slowdebug it fails as pkg is unused. Should this be removed or retained and marked explicitly so the compiler doesn't complain? I'm not sure whether this falls into the category of "Unused but useful to have in a debugger session" mentioned above